auth/user.go

```diff
@@ -509,15 +509,19 @@ func (user *userImpl) ExpandWildCardChannel(channels base.Set) base.Set {
 	return channels
 }
 
-func (user *userImpl) FilterToAvailableChannels(channels base.Set) ch.TimedSet {
-	output := ch.TimedSet{}
+func (user *userImpl) FilterToAvailableChannels(channels base.Set) (filtered ch.TimedSet, removed base.Set) {
```

Comment:
Can just return removed as []string to avoid the later conversion at logging time.